### DISCUSSION TITLE
Make website dark mode easier for the eyes

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#3423] [Ideas]
Website dark mode isn't very comfortable for the eyes. Here's why:
![Screenshot 2021-09-02 4 32 56 PM](https://user-images.githubusercontent.com/82263560/131873742-19a33659-5e44-479b-96d5-3cbad63cef37.png)
Post rescale button
![Screenshot 2021-09-02 4 33 09 PM](https://user-images.githubusercontent.com/82263560/131873759-4f182366-ff69-4454-becb-c015dd6b2728.png)
Post styling buttons
![Screenshot 2021-09-02 4 33 49 PM](https://user-images.githubusercontent.com/82263560/131873778-8df0c372-aba9-4173-a141-d707c313de9c.png)
Outline for About Me and WIWO
![Screenshot 2021-09-02 4 34 09 PM](https://user-images.githubusercontent.com/82263560/131873789-69bf53bd-1de0-4171-b1a6-4d77b7589d8a.png)
Studio thumbnail
![Screenshot 2021-09-02 4 34 45 PM](https://user-images.githubusercontent.com/82263560/131873814-5e15247f-9ad3-4383-a4a8-5034d04fe988.png)
Blue comment background

All of this combined makes website dark mode look like an eyesore.
I suggest fixing these, and possibly making website dark mode customisable in the next update. Thank you!

**Comment by VeryUsual** [comment]
I agree

**Comment by lisa-wolfgang** [comment]
I believe that customization for website dark mode specifically is on hold because we want to rework the entire theme system to be more unified.

**Reply by WorldLanguages** [reply]
I'm not sure

**Reply by mxmou** [reply]
Improvements to the theme system probably shouldn't change the way existing themes are implemented - there would still be a website dark mode addon with various settings, just the way people interact with it would be improved. For example, #2232 suggests using separate addons for different themes instead of using presets for a single addon.

So I don't think we need to wait until superpresets and things like that are implemented before adding new functionality to existing themes.
